src/views/Search.vue

```diff
@@ -137,6 +138,20 @@
       },
 
       methods: {
+
+        setTitle(results) {
+          if(results.length == 0) {
+            this.title = `Rechercher un financement : Aucun résultat pour «${this.text}» - France Relance - Ministère de la Transformation et de la Fonction publiques`;
```

Comment:
Mini corrections typographiques :

```suggestion
            this.title = `Rechercher un financement : Aucun résultat pour « ${this.text} » – France Relance – Ministère de la Transformation et de la Fonction publiques`;
```